0.Exploratory_Data_Analysis/README.md

```diff
@@ -0,0 +1 @@
+# This folder hosts jupyter notebooks for exploratory data analysis (EDA) for multiple 'waves' of data. See this repo's main page for a link to the Interstellar preprocessing page.
```

Comment:
Maybe it might be good to include what a `wave` is. Essentially describe that each wave is a different part of a 384 well plate that have different conditions, staining, etc.

I will/do reiterate this in image-based analysis repo, but it is always good to add the documentation in case someone doesn't go to the page or find it.